lua/telescope/actions/set.lua

```diff
@@ -124,6 +125,18 @@ set.edit = function(prompt_bufnr, command)
   end
 end
 
+--- Scrolls the previewer up or down
+---@param prompt_bufnr number: The prompt bufnr
+---@param direction number: The direction of the scrolling
+--      Valid directions include: 1 or -1
```

Comment:
FYI Multiline param currently not possible with `tree-sitter-lua` See https://github.com/tjdevries/tree-sitter-lua/pull/14#discussion_r587908659

Reply:
I should try that grammar as well, been wanting to try that but I always forgot :D

Reply:
Wouldn't recommend it quite yet. I am using a quite out of date `nvim-treesitter` to make it work.
I would wait until overriding a query is easier in `nvim-treesitter` and the queries for that grammar need some work too.

The only thing that you would gain from it is the emmy documentation part and you can play with `docgen` which is super cool :)